app/serializers/threshold_serializer.rb

```diff
@@ -0,0 +1,11 @@
+class ThresholdSerializer
+  def call(thresholds)
+    {
+      min: thresholds[:threshold_very_low],
+      low: thresholds[:threshold_low],
+      middle: thresholds[:threshold_medium],
+      high: thresholds[:threshold_high],
+      max: thresholds[:threshold_very_high],
```

Comment:
I think I would add this to stream serializer. We don't have any place where threshold data is needed independently. I think it will be easier to grasp when we have it in one file.